url/tests/urltestdata.json

```diff
@@ -7668,6 +7660,7 @@
     "hash": ""
   },
   "Do not serialize /. in path",
+  "skip next",
```

Comment:
I think we need to also explicitly handle this part [here](https://github.com/servo/rust-url/blob/edeaea7f1e8bd08148bcd535f921fbc645bce53e/url/src/parser.rs#L792-L794)